editorpythonbindings.md

```diff
@@ -0,0 +1,568 @@
+---
+description: ' Follow these steps to use or extend EditorPythonBindings. '
+title: Using & Extending EditorPythonBindings
+---
+
+EditorPythonBindings (EPB) offers access to build scalable multi-point, scriptable, end-to-end, round-trip pipelines, and common framework solutions for game content creators and Open 3D engine (O3DE) contributors via Python. These solutions will measurably drive things like:
+
+* Improve extensibility by reducing the barrier of entry for building out new custom workflows or pipelined processes
+* Increase efficiency via automation to reduce the number of clicks to operate
+* Allow automated content to process and run hands-free
+
+## Using EPB with Tests
+While the O3DE public documentation should serve as the primary source for usage, here is a brief example of how to use EPB in a test. 
+
+This code snippet the entry point into the entire object/widget hierarchy of our Qt application.  From here, you can do things like access the menu bars/toolbars, find children widgets to inspect/inject events onto, and more.
+
+```python
+import Tests.ly_shared.pyside_utils as pyside_utils
+ 
+# Returns a PySide2 wrapped instance of our QMainWindow
+editor_window = pyside_utils.get_editor_main_window()
+```
+
+As you can see, you can treat EPB as you can most other Python modules.
+
+## Extending EPB
+
+### Overview
+The EPB gem is an optional gem. This means that when a project (e.g. Starter Game or Samples Project) wants to expose scriptable functionality to Python, a project team needs to enable the “Editor Python Bindings” gem for the project. The next step is for the feature team to add behavior scoped for automation.
```

Comment:
Do we still have StarterGame and SamplesProject? I thought those were deprecated.